src/ocl/tensorocl.cpp

```diff
@@ -1896,6 +1896,10 @@ void CopyTensor(const Handle& handle,
     }
 }
 
+#ifdef __clang__
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wimplicit-fallthrough"
```

Comment:
We can use [`[[fallthrough]]`](https://en.cppreference.com/w/cpp/language/attributes/fallthrough)